aya/src/obj/btf/relocation.rs

```diff
@@ -558,31 +574,41 @@ impl<'a> AccessSpec<'a> {
                 }
             }
             RelocationKind::EnumVariantExists | RelocationKind::EnumVariantValue => match ty {
-                BtfType::Enum(en) => {
+                BtfType::Enum(_) | BtfType::Enum64(_) => {
```

Comment:
Maybe let's do this instead?

```suggestion
                BtfType::Enum(en) | BtfType::Enum64(en) => {
```

...

Reply:
what joshua said, this isn't valid rust